modules/pruning-images.adoc

```diff
@@ -25,20 +20,19 @@ When the Pruning Custom Resource is deleted, the pruning `CronJob` and its relat
 [source,yaml]
 ----
 spec:
-  schedule: "0 0 * * *" <1>
+  schedule: 0 0 * * * <1>
   suspend: false <2>
   keepTagRevisions: 3 <3>
-  keepYoungerThan: 60m <4>
+  keepYoungerThanDuration: 60m <4>
```

Comment:
In 4.4 we have keepYoungerThan that is an integer in nanoseconds. It's deprecated in 4.5 and the new field `keepYoungerThanDuration` with normal values (strings with suffixes, like `1h`, `60m`, or `1h30m`) should be used instead.

Should docs mention the deprecated field?

Reply:
That's a good question. I'm looking into it